source/parse.h

```diff
@@ -5899,8 +5903,12 @@ class parser
                 //  Next should be an expression-list followed by a ')'
                 //  If not, then this wasn't a call expression so backtrack to
                 //  the '(' which will be part of the next grammar production
+		        is_inside_call_expr = true;
 
                 term.expr_list = expression_list(term.op, lexeme::RightParen);
+				
+		        is_inside_call_expr = false;
+
```

Comment:
I thought, that it might be better to add `is_inside_call_expr` as an argument. But I think from the geeneral design it is not possible.  So the state modification is ok.  In order to make the more prominent, I would changed it to:

```suggestion

                is_inside_call_expr = true;  // Flag this expression list parse.
                term.expr_list = expression_list(term.op, lexeme::RightParen);
                is_inside_call_expr = false;

```

Reply:
That makes sense. Done.

Reply:
Right, it's either use out-of-band state, or wire a parameter through the call chain. Sometimes I do the former, sometimes the latter.